{
  unwindTest(new Callable<Integer>(){
    public Integer call() throws Exception {
      throw new MyException();
    }
  }
);
}
